core/src/main/java/org/mobilitydata/gtfsvalidator/notice/EmptyColumnNameNotice.java

```diff
@@ -19,14 +19,14 @@
 /**
  * A column name is empty. Such columns are skipped by the validator.
  *
- * <p>Severity: {@code SeverityLevel.WARNING}
+ * <p>Severity: {@code SeverityLevel.ERROR}
```

Comment:
This will ban 13 feeds in France, Canada and Japan. Google can make an exception for them.

Reply:
As suggested, this PR has been split into different PRs: one for each notice. As of today, we do not have the statistics yet for GTFS datasets, but will update when we have them.